main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
+from django.db.models import QuerySet
+
+
+def race_create(player_race_data: dict) -> QuerySet:
+    race_name = player_race_data["name"]
+    race, created = Race.objects.get_or_create(
+        name=race_name,
+        description=player_race_data["description"]
+    )
+    if created:
+        for skill_data in player_race_data["skills"]:
+            Skill.objects.create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race
+            )
+    return race
+
+
+def guild_create(guild_data: dict[str] = None) -> QuerySet | None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]